### ISSUE TITLE
Accessibility Testing for IVC Forms, Form 10-10d

### ISSUE THREAD

**Issue by rachaelpenfil** [department-of-veterans-affairs/va.gov-team#91133]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@rachaelpenfil

### Use of color and color contrast checks

- [ ] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

- On the [Applicant name and DOB](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-info) page, there is a color contrast issue. If you select the "add another applicant" button, the hint text under "Date of birth" does not meet contrast (the text reads: "For example: January 19 2000"). This error was discovered with Axe and is also documented in that section of this issue.

### Who completed the axe scans?

@rachaelpenfil

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

| Result | Page |
| -------- | ------- |
| No issues  | [Introduction](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d)   |
| No issues | [Signer information](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/signer-type)     |
| No issues    | [Signer mailing address](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/signer-mailing-address)   |
| No issues    | [Sponsor mailing address](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/sponsor-mailing-address)    |
| No issues    | [Sponsor identification information](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/sponsor-identification-info)    |
| No issues    | [Sponsor contact info](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/sponsor-contact-info)   |
| No issues    | [Signer contact info](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/signer-contact-info)  |
| No issues    | [Signer relationship to applicant](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/signer-relationship)   |
| No issues    | [Applicant name and DOB](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-info)   |
| No issues    | [Sponsor status](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/sponsor-status)   |
| No issues    | [Sponsor status details](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/sponsor-status-date)    |
| **1 issue: Elements must meet minimum color contrast ratio thresholds**    | [Applicant name and DOB](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-info)  |
| No issues    | [Applicant interstitial page](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-info-intro/0)   |
| No issues    | [Applicant Identification info](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-identification-info/0)    |
| No issues    | [Same mailing address?](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-mailing-same/0)   |
| No issues    | [New applicant mailing address](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-mailing-address/0)   |
| No issues    | [Applicant contact info](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-contact-info/0)   |
| No issues    | [Applicant sex listed at birth](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-gender/0)   |
| No issues    | [Applicant relationship to sponsor](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-relationship/0)   |
| No issues    | [Relationship to sponsor - child](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-relationship-child/0)   |
| No issues    | [Date of marriage to sponsor](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-marriage-date/0)   |
| No issues    | [If remarriage upload](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-marriage-upload/0)   |
| No issues    | [Birth certificate upload](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-relationship-child-upload/0)   |
| No issues    | [Proof of marriage](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-child-marriage-file/0)  |
| No issues    | [Medicare status](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-medicare/0)   |
| No issues    | [OHI status](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-other-insurance-status/0)   |
| No issues    | [Medicare Part D](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-medicare-continued/0)  |
| No issues    | [Upload Medicare Part A and B](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-medicare-upload/0)   |
| No issues    | [Part D Upload](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-medicare-d-upload/0)    |
| No issues    | [OHI upload](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-other-insurance-upload/1)    |
| No issues    | [OHI card Certification Upload](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-other-insurance-10-7959c-file/1)   |
| No issues    | [Dependent child age](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-dependent-status/1)    |
| No issues    | [Dependent upload](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-dependent-upload/1)   |
| No issues    | [Supporting files remaining](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/supporting-files)    |
| No issues    | [Supporting document summary](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/consent-mail)   |
| No issues    | [Review and submit](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/review-and-submit)   |
| No issues   | [Confirmation](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/confirmation)   |
| No issues    | [Sponsor info](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/sponsor-info   |
| No issues    | [Signer info](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/signer-info)   |
| No issues    | [Adopted child](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-child-adoption-file/0)    |
| No issues    | [School certification upload]   |
| No issues    | [Medicare ineligible](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-medicare-ineligible/)   |

### How did axe testing go?

3 issues. @baltes @michaelclement , we will need to resolve these before go live.

### Who completed the content zoom and reflow test?

@rachaelpenfil

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

- On the [Applicant information](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/applicant-info) page, the reflow of the error message when you try to submit without a complete DOB is strange, but still readable. This happens regardless of zoom level. Unsure if it happens on other pages as well.

### Who completed the keyboard navigation test?

@rachaelpenfil

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

No issues.

### Do you have any other results to share?

- On the [Signer Information](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/champva/apply-form-10-10d/signer-type) page:
     - The [Go to VA Form 21-0845 Authorization to Disclose Personal Information to a Third-Party (opens in new tab)](https://www.va.gov/find-forms/about-form-21-0845/] link opened in the same window, rather than a new tab
     - The [VA Form 21-22 Appointment of Veterans Service Organization as Claimant’s Representative (opens in new tab)](https://www.va.gov/find-forms/about-form-21-22/) link opened in the same window, rather than a new tab
     - The [VA Form 21-22a, Appointment of Individual as Claimant’s Representative (opens in new tab)](https://www.va.gov/find-forms/about-form-21-22a/) link opened in the same window, rather than a new tab
